recipes/busco/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin/
+cp BUSCO_v1.2.py $PREFIX/bin
+ln -s $PREFIX/bin/BUSCO_v1.2.py $PREFIX/bin/BUSCO_v1.1b1.py
```

Comment:
Yes, it would be possible. I forgot why I even created this hacky solution, instead of just installing BUSCO 1.1b1, but now I'm looking at it I remember that it has to with Python incompatibilities as 1.1b1 doesn't run on  2.7, which in turn is needed for some other dependencies. Anyway, I'll look into fixing this.